### PR TITLE
feat(esm): refactor for ESM modules naming - imports and circulars

### PR DESCRIPTION
**Pull request by iwarner**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -16,36 +24,14 @@
     "**/CVS": true
   },
   // Javascript
-  "javascript.validate.enable": false,
   "[javascript]": {
-    "editor.defaultFormatter": "numso.prettier-standard-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```
Comment (majid-amiri) [inline .vscode/settings.json:28]: @iwarner Why Typescript? Is this added intentionally?

File: babel.config.js [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
   plugins: [
     '@babel/plugin-transform-runtime',
     '@babel/plugin-proposal-class-properties',
-    [
-      'styled-components',
-      {
-        displayName: true,
-        ssr: true
-      }
-    ],
```
Comment (majid-amiri) [inline babel.config.js:12]: @iwarner Why remove SC? Isn't it better to keep it (does removing it makes any difference?)?

File: babel.config.js [hunk 3]
```diff
@@ -1,53 +1,63 @@
...
+        //   'styled-components',
+        //   {
+        //     // displayName: true,
+        //     // minify: true,
+        //     // namespace: 'cleverly',
+        //     // pure: true,
+        //     ssr: true
+        //     // transpileTemplateLiterals: true
+        //   }
+        // ]
```
Comment (majid-amiri) [inline babel.config.js:58]: Let's remove commented codes to keep this as clean as possible.

File: .vscode/settings.json (hunk 1 above)
Comment (iwarner) [reply .vscode/settings.json:28]: Not sure why that got added removed

File: babel.config.js (hunk 3 above)
Comment (iwarner) [reply babel.config.js:58]: still testing this so want to keep the comments in

Comment (majid-amiri) [reply babel.config.js:58]: gotcha

**Review by majid-amiri — APPROVED** [review]

**Pull request merged**